src/css/vars.css

```diff
@@ -131,3 +130,9 @@
   /* transitions */
   --ds-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, outline-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
 }
+
+@media screen and (max-width: 1023.5px) {
+  :root {
+    --navbar-height: calc(74 / var(--rem-base) * 1rem);
```

Comment:
Making the top nav height smaller on mobile widths gives more screen real estate for content. Especially helpful on phone screens

Reply:
We could probably improve this more by refactoring how the left nav is opened on mobile. The button for that currently takes up a whole row being in the "toolbar" element